lansa-vmss-windows-autoscale-sql-database/createUiDefinition.json

```diff
@@ -816,6 +842,7 @@
             "maxSizeBytes": "[steps('DatabaseConfig').maxSizeBytes]",
             "databaseLogin": "[steps('DatabaseConfig').databaseLogin]",
             "databaseLoginPassword": "[steps('DatabaseConfig').databaseLoginPassword]",
+            "publicIpNewOrExisting": "[steps('applicationGatewayConfig').publicIpAddress.newOrExistingOrNone]",
```

Comment:
Where is the parameter that you are referencing? `newOrExistingOrNone`
And why do you have a 'none' state? 
It looks like this has been put together in a rush.
12 days work?